src/member/member.ts

```diff
@@ -36,6 +36,7 @@ export type PublicProfile = {
 export enum MemberType {
```

Comment:
I don't know if it'll break something, but after checking the different names, as I understand, `Account` is the "smallest" unit to represent a user right ? Here I'd rename this enum if possible to 

```suggestion
export enum AccountType {
```